src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.js

```diff
@@ -202,7 +202,7 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
                                 containerStyle={styles.pr2}
                                 onPress={() => Navigation.navigate(ROUTES.SETTINGS_STATUS_CLEAR_AFTER_DATE)}
                                 errorText={customDateError}
-                                titleTextStyle={styles.flex1}
+                                titleStyle={styles.flex1}
```

Comment:
`titleTextStyle` was doing the same as `titleStyle` - see original `MenuItem` on main branch. So I treated it as a duplicate and renamed to `titleStyle`